spec/teacher_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe Teacher do
+  before :each do
+    @teacher = Teacher.new 24, 'Computer Science', 'Dami'
+  end
```

Comment:
- [ ] In addition to all the written tests, can you please write tests for this missing methods (e.g `can_use_services?`). 👍